src/components/Tooltip/index.tsx

```diff
@@ -28,21 +28,20 @@ interface TooltipProps {
   side?: TooltipPrimitive.TooltipContentProps['side']
   content?: React.ReactNode
   sideOffset?: number
-  className?: string
 }
 
 const Tooltip = React.forwardRef<
   React.ElementRef<typeof TooltipPrimitive.Content>,
   TooltipProps
->(({ children, side, content, sideOffset, className }, ref) => {
+>(({ children, side, content, sideOffset }, ref) => {
   return (
     <TooltipPrimitive.Provider>
-      <TooltipPrimitive.Root>
+      <TooltipPrimitive.Root delayDuration={100}>
```

Comment:
100 ms delay to open instead of the default 700. This prop can be passed to the provider instead, but according to the radix-ui docs the provider is more for site/app-wide configuration.